plugin/starwars.vim

```diff
@@ -1,6 +1,12 @@
+scriptencoding utf-8
+
 let s:dir = expand('<sfile>:h:h') . '/resources/'
 function! s:play(...) abort
-  let l:ep = get(a:000, 0, 1)
+  if has("patch-8.1.1453")
+    let l:ep = a:1
+  else
+    let l:ep = get(a:000, 0, 1)
+  endif
```

Comment:
What do you mean?

https://github.com/vim/vim/commit/3397f74ac2ac27f1eef48e950c3c8eeb0338fe55

Reply:
Sorry, I was misunderstood so I fixed.